### PR TITLE
VACMS 10018 queries refactor

### PR DESCRIPTION
**Pull request by timcosgrove**
## Description
WIP

### CODE REVIEW

File: src/templates/layouts/storyListingFull/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  title,
+  introText,
+  stories,
+}: StoryListingFullType) {
+  const storyTeasers = stories?.map((story: NewsStoryTeaserType) => (
+    <li key={story.id}>
+      <NewsStoryTeaser {...story} />
+    </li>
+  )) || (
+    <div className="clearfix-text">No stories at this time.</div>
```
Comment (timcosgrove) [inline src/templates/layouts/storyListingFull/index.tsx:27]: This might want to be `<li>`

File: src/data/delegators/Paragraph.tsx [hunk 2]
```diff
@@ -3,15 +3,15 @@ import {
...
-} from '@/types/paragraph'
-import { Meta as AudienceTopicsMeta } from '@/components/audience_topics/dataService'
-import { Meta as ButtonMeta } from '@/components/button/dataService'
-import { Meta as EmailContactMeta } from '@/components/email_contact/dataService'
-import { Meta as ExpandableTextMeta } from '@/components/expandable_text/dataService'
-import { Meta as StaffProfileMeta } from '@/components/staffProfile/dataService'
-import { Meta as LinkTeaserMeta } from '@/components/linkTeaser/dataService'
-import { Meta as WysiwygMeta } from '@/components/wysiwyg/dataService'
-import { Meta as RichTextCharLimit1000Meta } from '@/components/richTextCharLimit1000/dataService'
+} from '@/types/data-types/drupal/paragraph'
```
Comment (amponce) [inline src/data/delegators/Paragraph.tsx:6]: `data-types` to `dataTypes` ?

File: src/pages/demo/pagination/exampleA/[page].tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 import { GetStaticPathsResult } from 'next'
 import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
-import Container from '@/components/container'
-import { NewsStoryTeaser } from '@/components/newsStoryTeaser'
+import Container from '@/templates/common/container'
+import { NewsStoryTeaser } from '@/templates/components/newsStoryTeaser'
 import Pagination from '@department-of-veterans-affairs/component-library/Pagination'
-import { NodeResourceType } from '@/types/node'
-import { transformNewsStoryTeaserData } from '@/components/newsStoryTeaser/dataService'
+import { NewsStoryTeaserMapping } from '@/data/mappings/NewsStoryTeaserMapping'
```
Comment (amponce) [inline src/pages/demo/pagination/exampleA/[page].tsx:8]: newsStoryTeaserMapping if we're keeping the mapping directory

**Comment by penny-lischer** [comment]
@timcosgrove the src/components/header folder can be deleted

**Comment by amponce** [comment]
Updated the doc with some suggestions related to the directory current structure. Feel free to edit change or comment 🚧 

[Directory Structure Document ](https://docs.google.com/document/d/1afjQmF4ka3sSUCagmTYH0AWXxnjh-QQoHmxqf0A-NDM/edit)

Screen Shot 2022-08-11 at 1 29 06 PM

File: src/pages/demo/pagination/exampleA/[page].tsx (hunk 3 above)
Comment (timcosgrove) [reply src/pages/demo/pagination/exampleA/[page].tsx]: Removing mappings; also, we need to revisit these pagination examples.

**Review by amponce — APPROVED** [review]
LGTM 👍

**Pull request merged**